### DISCUSSION TITLE
Axis Calibrator.

### DISCUSSION THREAD

**Discussion by StuartB4** [arkypita/LaserGRBL#2216] [Ideas]
Hi Arkypita.

Although I mainly use LaserGRBL, and have since it was first released, I also use LightBurn for designing.
I noticed in Machine Settings they added an Axis Calibrator for steps/mm.  
I just wondered if you had thought about adding one to laserGRBL.  

LightBurn Image. 
![Calibrate](https://github.com/arkypita/LaserGRBL/assets/16980026/14dfeee2-5700-49b0-8cb2-fbc3942ba631)

**Comment by arkypita** [comment]
Hi @StuartB4 
Yes, axis calibrator is already in my mind.
Easy to implement and useful, i think you will se it soon.

**Comment by StuartB4** [comment]
Oh that is good. I still get a few people asking for the best way to calibrate.
Thanks very much for the reply, I appreciate it. :)
